benchmark/single-source/StringRepeating.swift

```diff
@@ -0,0 +1,83 @@
+//===--- StringRepeating.swift -------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import TestsUtils
+
+public let benchmarks = [
+  BenchmarkInfo(name: "String.initRepeating.1AsciiChar.Count100",
+                runFunction: run_singleAsciiCharacterCount100,
+                tags: [.validation, .api, .String]),
+  BenchmarkInfo(name: "String.initRepeating.26AsciiChar.Count2",
+                runFunction: run_26AsciiCharactersCount2,
+                tags: [.validation, .api, .String]),
+  BenchmarkInfo(name: "String.initRepeating.33CyrillicChar.Count2",
+                runFunction: run_33CyrillicCharactersCount2,
+                tags: [.validation, .api, .String]),
+  BenchmarkInfo(name: "String.initRepeating.longMixedString.Count100",
+                runFunction: run_longMixedStringCount100,
+                tags: [.validation, .api, .String])
+]
+
+@inline(never)
+func repeating(_ i: String, count: Int) -> String {
+  let s = String(repeating: getString(i), count: count)
+  return s
+}
+
+@inline(never)
+public func run_singleAsciiCharacterCount100(N: Int) {
+    for _ in 1...5000*N {
+        blackHole(repeating("x", count: 100))
+    }
+}
+
+@inline(never)
+public func run_26AsciiCharactersCount2(N: Int) {
+    for _ in 1...5000*N {
+        blackHole(repeating("abcdefghijklmnopqrstuvwxyz", count: 2))
+    }
+}
+
+@inline(never)
+public func run_33CyrillicCharactersCount2(N: Int) {
+    for _ in 1...5000*N {
+        blackHole(repeating("абвгґдеєжзиіїйклмнопрстуфхцчшщьюя", count: 2))
+    }
+}
+
+func getLongString() -> String {
+  let long = """
+    Swift is a multi-paradigm, compiled programming language created for
```

Comment:
This looks like copyrighted text. Did you secure permission to re-license this here? If not, please find public domain multilingual text. I _believe_ the Universal Declaration of Human Rights is one example but you would need to double-check.

Reply:
This exact string appears in following files in this repo:
* `benchmark/single-source/CString.swift`
* `benchmark/single-source/UTF8Decode.swift`
* `benchmark/single-source/StringBuilder.swift`
* `benchmark/single-source/StringSplitting.swift`
* `validation-test/compiler_crashers_2_fixed/0109-issue-47314.swift`
* `test/Prototypes/UnicodeDecoders.swift`
This suggests it should be safe to use this string once again.